src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -38,7 +38,9 @@
     protected $crawler;
     protected $insulated = false;
     protected $redirect;
+    protected $redirects = array();
     protected $followRedirects = true;
+    protected $browserNavigation = false;
```

Comment:
please use private visibility for any new property. We don't want to make them extension points yet